README.md

```diff
@@ -9,15 +9,21 @@ Official React Native bindings for Bitmovin's mobile Player SDKs.
 
 > :warning: **Beta Version**: The library is under active development. The current Beta release supports basic playback of unprotected video assets.
 
-- [Installation](#installation)
-  - [Add package dependency](#add-package-dependency)
-  - [Setup iOS Player SDK](#setup-ios-player-sdk)
-  - [Setup Android Player SDK](#setup-android-player-sdk)
-- [Getting Started](#getting-started)
-  - [Setting up a license key](#setting-up-a-license-key)
-  - [Accessing native `Player` instances](#accessing-native-player-instances)
-  - [Listening to events](#listening-to-events)
-- [Contributing](#contributing)
+- [Bitmovin Player React Native](#bitmovin-player-react-native)
+  - [Installation](#installation)
+    - [Add package dependency](#add-package-dependency)
+    - [Setup iOS Player SDK](#setup-ios-player-sdk)
+    - [Setup Android Player SDK](#setup-android-player-sdk)
+  - [Getting Started](#getting-started)
+    - [Setting up a license key](#setting-up-a-license-key)
+      - [Configuring through code](#configuring-through-code)
+    - [Setting up a playback configurations](#setting-up-a-playback-configurations)
+      - [Configuring through code](#configuring-through-code-1)
+      - [Configuring `Info.plist`](#configuring-infoplist)
+      - [Configuring `AndroidManifest.xml`](#configuring-androidmanifestxml)
```

Comment:
Those two topics are related to the `Setting up a license key` section, not the playback configuration.

Reply:
Please, move them back to the `Setting up a license key` section just the way it was.